source/whats-new.txt

```diff
@@ -32,6 +33,21 @@ Learn what's new in:
 
 .. _upcoming-breaking-changes:
 
+.. _csharp-version-2.29:
+
+What's New in 2.29
+------------------
+
+The 2.29 driver release has been tested against {+mdb-server+} version 8.0 and
+includes the following new features:
+
+- Adds support for v2 of the Queryable Encryption range protocol.
+
+- Adds support for range indexes for Queryable Encryption.
```

Comment:
Add a link to more info about QE: see https://www.mongodb.com/docs/drivers/node/current/whats-new/#std-label-version-6.9:~:text=Adds%20official%20support%20for%20Queryable%20Encryption%20(QE)%20range%20queries.%20To%20use%20this%20feature%2C%20your%20app%20must%20use%20mongodb%2Dclient%2Dencryption%20v6.1.0%20or%20later%20and%20must%20connect%20to%20MongoDB%20Server%208.0%20or%20later.%20For%20more%20information%20about%20QE%20range%20queries%2C%20see%20Queryable%20Encryption%20in%20the%20MongoDB%20Server%20manual.